.github/workflows/ci.yaml

```diff
@@ -75,7 +85,29 @@ jobs:
       fail-fast: false
       matrix:
         groups: ${{ fromJSON(needs.gh-hosted-collect-integration-tests.outputs.groups) }}
-    name: (GH hosted) ${{ matrix.groups.job_name }}
+        juju-snap-channel:  ["2.9/stable", "3.1/candidate"]
+        include:
+          - juju-snap-channel: "3.1/candidate"
+            agent-version: "3.1.6"
+            libjuju-version:  "3.2.0.1"
+          - juju-snap-channel: "2.9/stable"
+            agent-version: "2.9.44.0"
+            libjuju-version:  "2.9.44.0"
```

Comment:
please bump to `2.9.44.1` as `.0` will break with python 3.10 (default on jammy)